contracts/ServiceNodeContribution.sol

```diff
@@ -225,7 +263,8 @@ contract ServiceNodeContribution is Shared {
      * be calculated by calling `calcMinimumContribution` with the staking
      * requirement and 0 contributors.
      */
-    function resetContract(uint256 amount) external onlyOperator {
+    function resetContract(uint256 amount, IServiceNodeRewards.Contributor[] memory reservedContributors) external onlyOperator {
```

Comment:
Perhaps `= []` default on reservedContributors, if that's something solidity allows.

Reply:
See comment on other. But not available in solidity